Gemfile

```diff
@@ -70,3 +72,5 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+
+gem "rack-cors"
```

Comment:
- [ ] As earlier suggested. It will be nice to add the `webpack` and `react-rails` gems. You need the `webpack` as your javascript bundler. And the `react-rails` gems so the react app can integrate and work with rails. So kindly add the gems to the `gemfile` and run `bundle install` to install them into the app.

Reply:
Hey @enaburekhan  

- You are saying 
It will be nice to also add the webpack and react-rails gems ❌ (Not the preferred way of Rails 7)
You need the webpack as your javascript bundler.  ❌  (Not the only option or the best option)
And the react-rails gems so the react app can integrate and work with rails. ❌  (There are no need for these gems to integrate)
So kindly add the gems to the gemfile and run bundle install to install them into the app. ❌  (Repeated)

> - Check this part  [Rails 7.0: Fulfilling a vision](https://rubyonrails.org/2021/12/15/Rails-7-fulfilling-a-vision#:~:text=Gone%20is%20the%20tightly%2Dcoupled%20embrace%20of%20Webpack%20with%20Webpacker%2C%20and%20instead%20is%20a%20loosely%2Dcoupled%20embrace%20of%20any%20JavaScript%20bundler) to help you with what I am talking about!

Reply:
Check that also [But today the trade-offs made for Webpacker are starting to make a lot less sense.] (https://world.hey.com/dhh/rails-7-will-have-three-great-answers-to-javascript-in-2021-8d68191b#:~:text=But%20today%20the%20trade%2Doffs%20made%20for%20Webpacker%20are%20starting%20to%20make%20a%20lot%20less%20sense.) will help to understand the big picture of Rails 7